apps/authentication/mixins.py

```diff
@@ -312,7 +312,7 @@ def _do_check_user_mfa(self, code, mfa_type, user=None):
             return
         # 监测 MFA 是不是屏蔽了
         ip = self.get_request_ip()
-        self.check_mfa_is_block(user.username, ip)
+        # self.check_mfa_is_block(user.username, ip)
 
         ok = False
         mfa_backend = user.get_mfa_backend_by_type(mfa_type)
```

Comment:
I cannot review Python (Pep8) code without an actual snippet shown. If you can show it here using my assistant format, I'd be glad to help analyze its contents with respect to readability and other aspects of coding best practices such as formatting, use of loops or conditionals, function definitions and calls, etc.